ansible/roles/products/common/arm-interfaces/templates/netplan-left.j2

```diff
@@ -0,0 +1,15 @@
+#jinja2: trim_blocks:False
+network:
+    ethernets:
+        {{ ethercat_left_arm }}:
+            addresses: [192.168.2.100/24]
+            optional: true
+            routes:
+                - to: 192.168.2.1/32
+                  via: 192.168.2.1
```

Comment:
No, it will be configured as 192.168.2.1 because: https://github.com/shadow-robot/sr-config/blob/melodic-devel/sr_ur_arm_config/config/ur_arm_2_robot_hw.yaml

Reply:
ah cool, great we changed our config